### ISSUE TITLE
Add `libprotobuf-dev` and `protobuf-compiler` to dependencies list

### ISSUE THREAD

**Issue by bogdasar1985** [p4lang/p4c#2856]
In my system(Linux Mint 20) CMake output next:
```
CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find Protobuf (missing: Protobuf_INCLUDE_DIR) (Required is at
  least version "3.0.0")
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.16/Modules/FindProtobuf.cmake:624 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:110 (find_package)
```

It fixed by adding a `libprotobuf-dev` to dependencies list

**Comment by mbudiu-vmw** [comment]
Can you try to submit a PR with this change?

**Comment by bogdasar1985** [comment]
@mbudiu-vmw When I said "dependencies list" I meant just the bash script from README.md if you want to test it on CI/CD, then I don't think anything will change

**Comment by fruffy** [comment]
`libprotoc-dev` and  `protobuf-compiler` are only available starting from Ubuntu 18.04. We still support Ubuntu 16.04, so this does not generalize. This note should be included if we add it to the README. Also, anything we say in the README should ideally be tested by CI.
